test/helpers/interfaceArtifacts.js

```diff
@@ -40,6 +40,7 @@ export const interfaceToArtifacts = {
   [keyToKey.equityTokenControllerInterface]: [artifacts.PLACEHOLDER_EQUITY_TOKEN_CONTROLLER, ""],
   [keyToKey.termsInterface]: [artifacts.STANDARD_ETO_TERMS, ""],
   [keyToKey.paymentTokenInterface]: ["IERC223Token"],
+  [keyToKey.etoTermsConstraints]: [artifacts.STANDARD_ETO_TERMS, ""],
```

Comment:
really `STANDARD_ETO_TERMS`? we should have new artifact for ETOConstraints in config!

Reply:
yeah, that's a typo! done